### PR TITLE
Adding FactorialHR company

### PR DESCRIPTION
**Pull request by gtrias**
### Motivation

In FactorialHR we've been using Sorbet since Jun 27, 2019. So it would be nice to be in the list of users using this amazing library :)

### CODE REVIEW

**Comment by elliottt** [comment]
Screenshot of the changes:

Screen Shot 2020-04-16 at 10 06 17 AM

File: website/siteConfig.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ const users = [
     image: 'img/vonage-logo.png',
     infoLink: 'https://www.vonage.com/',
   },
+  {
+    caption: 'FactorialHR',
+    image: 'img/factorial-logo.png',
+    infoLink: 'https://www.factorialhr.com/',
+  },
```
Comment (elliottt) [inline website/siteConfig.js:47]: Could you move this entry to right after `Chan Zuckerberg Initiative` (line 28)?

Comment (gtrias) [reply website/siteConfig.js:47]: sure!

Comment (gtrias) [reply website/siteConfig.js]: done :)

**Comment by gtrias** [comment]
> Thanks!

Thank you!

**Pull request merged**